### ISSUE TITLE
k-9 drains battery? (cf #1290)

### ISSUE THREAD

**Issue by mh-g** [k9mail/k-9#1332]
Re-issuing item #1290:

### Expected behaviour
K-9 Mail should not use more battery than other apps.

### Actual behaviour
K-9 Mail drains more battery than any other app, by a factor of >4. In fact, K-9 Mail uses more battery than the rest of the system combined.

### Steps to reproduce
1. Just wait ...
2. Actions from #857 do NOT help. Whitelisting K-9 Mail still has enormous battery drain.

### Environment
K-9 Mail version:
5.010

Android version:
6.0.1 (CyanogenMod)

Account type (IMAP, POP3, WebDAV/Exchange):
3x IMAP, 1x POP3

As a comparison, I will keep an inofficial port of trojita on one of the IMAP accounts up and running for the next day.

**Comment by ocdtrekkie** [comment]
This issue is almost useless without a description of what your syncing settings are for each account that you use. If you have your email manually sync every 60 seconds, it's going to use a lot more battery than say, push mail or an hourly sync or what have you.

**Comment by mh-g** [comment]
Sync settings were set to "never". I changed them to once per hour for the next test. Maybe "never" was implemented wrongly?

**Comment by mh-g** [comment]
My wife has the same version on 4.4 (also CyanogenMod), one account with the same provider and the same settings. Battery drain is a tenth below the top drainer (i.e. about a 1/50-1/100 of what I experience on Android 6.0.1).

**Comment by philipwhiuk** [comment]
> Synchronization related battery issues related to 6.0.1 are discussed here: #970

Please read what I wrote closing it originally and don't re-open a duplicate.

**Comment by golferguy** [comment]
Wanted to pass this on to K9 team.  K9 mail is definitely a battery killer.
Phone 1:  Samsung Note 4 Lollipop 
Phone 2: LG G4 with Marshmallow

Both phones have 2 accounts each on outlook.com
Phones were set to poll every 15 minutes due to lack of real IMAP Push or Activesync support.

After one year of use of K9 and batteries lasting about 8 hours, switched both phones from K9 mail to outlook.com apps.

Battery life on both phones is now over 12 hours each.  50% increase since polling isn't needed.

I miss some of the K9 features but the battery life difference is 50%.  Hope K9 can avoid need for polling.

**Comment by gdt** [comment]
@golferguy: 8 vs 12 is hard to follow given that there is no info on what else is happening, and how much time the radio is active, bytes sent, CPU used, etc.   But to make a fair comparison you should set the other app to poll every 15 also.    You said "lack of real IMAP push".   I run K-9, and with little else running, my phone will go 2 days - but that's with a mail server that does IMAP idle correctly.   K-9 does IMAP idle just fine, and if outlook.com doesn't support open standards you should file a bug report with them :-)
